config/routes.rb

```diff
@@ -254,6 +253,12 @@
 
     resources :event_kinds, module: 'event', controller: 'kinds'
 
+    scope module: :mailing_lists do
```

Comment:
```suggestion
    scope '/mailing_lists', module: :mailing_lists do
```
then remove mailing_lists from get, delete ...